guide.md

```diff
@@ -45,45 +34,139 @@ We designed this guide to help anyone navigate those complexities—regardless o
+
+<div class="columns-desktop -two -ref">
+<div class="column" markdown="1">
+
+{% include picture.html
+   image = "/assets/images/guide/savings-account.png"
+   retina = "/assets/images/guide/savings-account@2x.png"
+   alt-text = "Strong handshake"
+   width = 400
+   height = 210
+%}
+
+<h2 class="h3" markdown="1">[Savings wallet]({{ '/guide/case-studies/savings-account/' | relative_url }})</h2>
+
+A multi-key mobile wallet design that provides great security for larger amounts, and low-friction for smaller transactions.
 
-Familiarizes designers with the products, features, and terms needed to create better experiences for first time users.
+</div>
+<div class="column" markdown="1">
+
+{% include picture.html
+   image = "/assets/images/guide/upgradeable-account.png"
+   retina = "/assets/images/guide/upgradeable-account@2x.png"
+   alt-text = "Strong handshake"
+   width = 400
+   height = 210
+%}
+
+<h2 class="h3" markdown="1">[Upgradeable account]({{ '/guide/case-studies/upgradeable-wallet/' | relative_url }})</h2>
+
+Exploring the concept of progressive security, in which applications help users navigate varying security configurations appropriate to their holdings and use case.
+
+</div>
+</div>
+
+<div class="columns-desktop -two -ref">
+<div class="column" markdown="1">
+
+{% include picture.html
+   image = "/assets/images/guide/shared-accounts.png"
+   retina = "/assets/images/guide/shared-accounts@2x.png"
+   alt-text = "Strong handshake"
+   width = 400
+   height = 210
+%}
+
+<h2 class="h3" markdown="1">[Shared accounts]({{ '/guide/case-studies/shared-account/' | relative_url }})</h2>
+
+An overview and considerations for bitcoin wallets that are managed by multiple owners.
+
+</div>
+<div class="column -blank" markdown="1">
+
+</div>
+</div>
 
 ---
 
-### [Private key management]({{ '/guide/private-key-management/introduction/' | relative_url }})
+<h2 class="h3" markdown="1">[How it works]({{ '/how-it-works/' | relative_url }})</h2>
+
+Explanations of various technical aspects of bitcoin and Lightning.
+
+<div class="columns-desktop -two -how">
+<div class="column" markdown="1">
+
+[Private key management]({{ '/guide/private-key-management/introduction/' | relative_url }})<br />
+[Automatic cloud backup]({{ '/guide/private-key-management/cloud-backup/' | relative_url }})<br />
+[Manual backup]({{ '/guide/private-key-management/manual-backup/' | relative_url }})<br />
+[External signers]({{ '/guide/private-key-management/external-signers/' | relative_url }})<br />
+[Multi-key]({{ '/guide/private-key-management/multi-key/' | relative_url }})
```

Comment:
```suggestion
[Private key management]({{ '/guide/how-it-works/private-key-management/introduction/' | relative_url }})<br />
[Automatic cloud backup]({{ '/guide/how-it-works/private-key-management/cloud-backup/' | relative_url }})<br />
[Manual backup]({{ '/guide/how-it-works/private-key-management/manual-backup/' | relative_url }})<br />
[External signers]({{ '/guide/how-it-works/private-key-management/external-signers/' | relative_url }})<br />
[Multi-key]({{ '/guide/how-it-works/private-key-management/multi-key/' | relative_url }})
```

Updated links

Reply:
Link changes should be part of the update from master. Otherwise I'd be merging in URLs that don't exist in the branch yet.